PayloadPkg/OsLoader/OsLoader.c

```diff
@@ -552,6 +552,9 @@ BeforeOSJump (
   }
```

Comment:
pls put Payload changes in a separate commit.

Reply:
will do it.